ood-portal-generator/spec/fixtures/ood-portal.conf.maint_with_ips

```diff
@@ -78,7 +78,7 @@
   #
   SetEnv OOD_PUN_STAGE_CMD "sudo /opt/ood/nginx_stage/sbin/nginx_stage"
 
-  SetEnv OOD_ALLOWED_HOSTS "example.com"
+  SetEnv OOD_ALLOWED_HOSTS "8.8.8.8,example.com"
```

Comment:
It appears that `@proxy_server` is _never_ nil because it's populated by the fqdn. So that's why `example.com` almost always shows up here in `OOD_ALLOWED_HOSTS`.